plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/sql/SQLUtils.java

```diff
@@ -221,6 +221,21 @@ public static String makeSQLLike(String like)
         return like.replace("*", "%").replace("?", "_");
     }
 
+    public static String makeGlobFromSqlLikePattern(String sqlLikePattern) {
+
+        StringBuilder result = new StringBuilder();
+        for (int i = 0; i < sqlLikePattern.length(); i++) {
+            char charAtI = sqlLikePattern.charAt(i);
+            switch (charAtI) {
+                case '_' -> result.append('?');
+                case '%' -> result.append('*');
+                case '?' -> result.append("\\?");
+                default -> result.append(charAtI);
+            }
```

Comment:
You also need to escape the `*` character

Reply:
Yes, there is. Since both `?` and `*` are special markers when in the glob, you want to escape all occurrences of these in the original string. You escape `?` but not `*`, why?

Reply:
![image](https://github.com/dbeaver/dbeaver/assets/43766501/81f5a543-c283-4bbc-a903-44b7205503a8)
That's why